tdp/cli/commands/deploy.py

```diff
@@ -62,6 +64,12 @@
 )
 @click.option("--filter", type=str, help="Glob on list name")
 @click.option("--dry", is_flag=True, help="Execute dag without running any operation")
+@click.option(
+    "--resume",
+    type=str,
+    metavar="dep_id",
+    help="Resume a previous deployment, cannot be used with --sources or --targets",
```

Comment:
`--filter` is allowed ? If the "resumed deployment" had a `filter`, the `--filter` will be overridden. I think `--filter` cannot be used with `resume` too.